pandas/tests/extension/test_arrow.py

```diff
@@ -3051,3 +3052,13 @@ def test_string_to_datetime_parsing_cast():
         ArrowExtensionArray(pa.array(pd.to_datetime(string_dates), from_pandas=True))
     )
     tm.assert_series_equal(result, expected)
+
+
+def test_string_to_time_parsing_cast():
+    # GH 56463
+    string_times = ["11:41:43.076160"]
+    result = pd.Series(string_times, dtype="time64[us][pyarrow]")
+    expected = pd.Series(
+        ArrowExtensionArray(pa.array(to_time(string_times), from_pandas=True))
```

Comment:
can you make the to_time result more explicit; i dont think to_time is well-maintained and dont particularly trust it

Reply:
Good idea. Using a `datetime.time` object here makes more sense to test